src/main/java/Application.java

```diff
@@ -0,0 +1,62 @@
+package main.java;
+
+import java.util.Scanner;
+
+class Operations {
```

Comment:
It's really really not good to have more then one class in one file. So please, separate it. 

The only one allowed case is when you use static inner classes. But here is not the case.